doc/nrf/releases_and_maturity/releases/release-notes-2.8.0.rst

```diff
@@ -0,0 +1,1319 @@
+
+* Added:
+
+  * :ref:`Compressed firmware update support <mcuboot_image_compression>`.
+
+* Fixed:
+
+  * An issue with wrong network core address checking in MCUboot for nRF5340 network core firmware updates.
+  * An issue with MCUboot updates in MCUboot when a swap mode was selected which could lead to possible data corruption by implementing additional NSIB-specific (overwrite-only) upgrade mode for sysbuild.
+  * An multiple issues with checking firmware update slot selection in MCUboot for sysbuild.
+  * An issue with MCUboot cleanup unused slots feature which could lead to possible empty firmware update of the main image and bricking the device for sysbuild.
+
+Zephyr
+======
+
+.. NOTE TO MAINTAINERS: All the Zephyr commits in the below git commands must be handled specially after each upmerge and each nRF Connect SDK release.
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``ea02b93eea35afef32ebb31f49f8e79932e7deee``, with some |NCS| specific additions.
+
+For the list of upstream Zephyr commits (not including cherry-picked commits) incorporated into nRF Connect SDK since the most recent release, run the following command from the :file:`ncs/zephyr` repository (after running ``west update``):
+
+.. code-block:: none
+
+   git log --oneline ea02b93eea ^23cf38934c
+
+For the list of |NCS| specific commits, including commits cherry-picked from upstream, run:
+
+.. code-block:: none
+
+   git log --oneline manifest-rev ^ea02b93eea
+
+The current |NCS| main branch is based on revision ``ea02b93eea`` of Zephyr.
+
+.. note::
+   For possible breaking changes and changes between the latest Zephyr release and the current Zephyr version, refer to the :ref:`Zephyr release notes <zephyr_release_notes>`.
+
+Additions specific to |NCS|
+---------------------------
+
+|no_changes_yet_note|
+
+Trusted Firmware-M
+==================
+
+* Added possibility to read UICR.OTP registers through platform services.
+
+Documentation
+=============
+
+* Added:
+
+  * The :ref:`ug_app_dev` section, which includes pages from the :ref:`configuration_and_build` section and from the removed Device configuration guides section.
+  * The :ref:`dfu_tools_mcumgr_cli` page after it was removed from the Zephyr repository.
+  * The :ref:`ug_nrf54h20_suit_soc_binaries` page.
+  * The :ref:`ug_nrf54h20_suit_push` page documenting the SUIT push model-based update process.
+  * The :ref:`ug_nrf54h20_suit_recovery` page.
+  * The :ref:`nrf_rpc_uart` page.
+  * The :ref:`ug_bt_stack_arch` and the :ref:`ug_bt_solution` documentation to the :ref:`Bluetooth protocols <ug_bt>` page.
+  * The :ref:`nfc_rpc` page documenting the NFC Remote Procedure Call (RPC) solution.
+  * The :ref:`memory_storage` page under :ref:`ug_nrf54l`.
+
+* Updated:
+
+  * The :ref:`ug_nrf70_developing_debugging` page with the new snippets added for the nRF70 driver debug and WPA supplicant debug logs.
```

Comment:
The first link (1) in this section and the last one (2) are referring to very specific pages within the structure, but the rendered name suggests general pages:
![image](https://github.com/user-attachments/assets/70f891ef-0ff9-4fa6-9c45-215786e2aabc)

For 1), can we add `mcumgr` to _the title of the page_? Also to avoid Kapa and search potentially linking to this page whenever someone searches for "command line"?

For 2) can we add a mention of the nRF7002 DK guide?
```
  * The nRF7002 DK device guide's :ref:`ug_nrf70_developing_debugging` page with the new snippets added for the nRF70 driver debug and WPA supplicant debug logs.
```